doozer/tests/test_metadata.py

```diff
@@ -188,22 +190,22 @@ def list_builds(packageID=None, state=None, pattern=None, queryOpts=None, **kwar
         self.assertEqual(meta.get_latest_build(default=None), builds[3])
 
         # Make sure that just matching the prefix of an assembly is not sufficient.
```

Comment:
I looked and this case basically is not supported with the code we have right now ("ignore a very similar looking assembly's build"). We fallback on finding stream build then which isn't too bad but not ideal. Would categorize this as a regression and discuss what we can do to support this. Maybe go back to fetching multiple builds.

Reply:
One idea is to strictly look for ending "." in ".assembly.<name>.elX" since now we always have "elX"

Reply:
using this now